src/Auth0.ManagementApi/Models/RefreshToken.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Auth0.ManagementApi.Models
+{
+    /// <summary>
+    /// Represents configuration of refresh tokens for a client.
+    /// </summary>
+    public class RefreshToken
+    {
+        /// <summary>
+        /// Refresh token rotation type
+        /// </summary>
+        [JsonProperty("rotation_type")]
+        [JsonConverter(typeof(StringEnumConverter))]
+        public RefreshTokenRotationType RotationType { get; set; } = RefreshTokenRotationType.NonRotating;
+
+        /// <summary>
+        /// Refresh token expiration type
+        /// </summary>
+        [JsonProperty("expiration_type")]
+        [JsonConverter(typeof(StringEnumConverter))]
+        public RefreshTokenExpirationType ExpirationType { get; set; } = RefreshTokenExpirationType.NonExpiring;
```

Comment:
I don't think the above default values are a good idea. They are the least secure options and I do not think we should use them as default values.

I would remove any default but instead enforce the need to specify both of these properties when passing RefreshToken.